packages/webdriverio/src/utils/detectBackend.ts

```diff
@@ -97,6 +97,19 @@ export default function detectBackend(options: BackendConfigurations = {}) {
         }
     }
 
+    /**
+     * Lambdatest
+     * e.g. cYAjKrqGwyPjPQv41ICDF4C5OjlxzA9epZsnugVJJxqOReWRWU
+     */
+    if (typeof user === 'string' && typeof key === 'string' && key.length === 50) {
```

Comment:
Not a concert regarding the PR but it doesn't seem to be a good idea to rely on key length.
Maybe it makes sense to explicitly set a service type? 
cc: @christian-bromann

Reply:
@mgrybyk it is currently the only way to differentiate between vendors. We do the same for others too. At the end of the day you can always provide a target host to connect to.